README.md

```diff
@@ -358,10 +365,77 @@ run the application. Press the start button and say
 
 For the full set of supported commands refer to [demo's readme](/demo/android/README.md).
 
-### JavaScript Demos
+### Web Demos
 
-There is a ["Vanilla" JavaScript demo](./demo/javascript/vanilla) and [React demo](./demo/javascript/react) available,
-both of which [run offline in the browser](https://picovoice.ai/blog/offline-voice-ai-in-a-web-browser/).
+From [demo/web](/demo/web) run the following in the terminal:
+
+```yarn
+yarn
+yarn start
+```
+
+(or)
+
+```
+npm install
+npm run start
+```
+
+Open http://localhost:5000 in your browser to try the demo.
+
+#### Angular Demos
+
+From [demo/angular](/demo/angular) run the following in the terminal:
+
+```yarn
+yarn
+yarn start
+```
+
+(or)
+
+```
+npm install
+npm run start
+```
+
+Open http://localhost:4200 in your browser to try the demo.
```

Comment:
we should show the YAML. Possibly grab it programmatically. It might help to increase wake word sensitivity

also when the wake word is detected we should clear the last inference.